core/domain/html_cleaner.py

```diff
@@ -18,9 +18,12 @@
 
 import logging
 import urlparse
+from HTMLParser import HTMLParser
+import json
 
 import bleach
 
+from bs4 import BeautifulSoup
```

Comment:
Generally we import at module level and refer to things using module.class, e.g. bleach.clean. Can we do this here (import bs4)? This import should also go in the same "import group" as `import bleach`.